langchain4j/src/main/java/dev/langchain4j/store/embedding/chroma/ChromaApi.java

```diff
@@ -0,0 +1,28 @@
+package dev.langchain4j.store.embedding.chroma;
+
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+
+interface ChromaApi {
+
+    @GET("/api/v1/collections/{collection_name}")
+    @Headers({"Content-Type: application/json"})
+    Call<Collection> collection(@Path("collection_name") String collectionName);
+
+    @POST("/api/v1/collections")
+    @Headers({"Content-Type: application/json"})
+    Call<Collection> createCollection(@Body CollectionCreationRequest collectionCreationRequest);
```

Comment:
minor: maybe better to rename CollectionCreationRequest into CreateCollectionRequest?